annotation/tests/test_distribution.py

```diff
@@ -1277,3 +1283,191 @@ def test_get_page_number_combinations(
     expected: List[int],
 ) -> None:
     assert get_page_number_combinations(file_pages, user_pages) == expected
+
+
+# ADDITIONAL TESTS
+
```

Comment:
We should check if similar mock objects already exist, so that we do not create unnecessary new objects.

Reply:
Mocked the SPLIT_MULTIPAGE_DOC var and simplified the test